1/Dockerfile

```diff
@@ -48,6 +48,35 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod -R 777 /etc/alternatives && \
+    chmod -R 777 /var/lib/alternatives && \
+    chmod -R 777 /usr/lib/jvm && \
+    chmod 777 /usr/bin && \
+    chmod 777 /usr/lib/jvm-exports && \
+    chmod 777 /usr/share/man/man1 && \
```

Comment:
turns out the group only has r-x permissions:

```
drwxr-xr-x.  2 root root 4096 Jan 25 19:01 man1
```

Leaving things as they are wrt chmod, unless I'm misinterpreting your previous comment.